server.py

```diff
@@ -325,7 +325,8 @@ def run_addfields_data():
         typeVal = query_dict.pop('type')
 
     if typeVal == "bug":
-        sql = "update alerts set bug='%s',status='Investigating' where id=%s;" %(data['BugID'], data['id'])
+        ids = data['id'][:-1]
+        sql = "update alerts set bug='%s',status='Investigating' where id IN(%s);" %(data['BugID'], ids)
```

Comment:
using the IN operator, do we need to add any type of quotes around the %s ?
Also what if data['id'] == '' ?